quickref/qsharp-quick-reference.tex

```diff
@@ -644,8 +644,8 @@
   \section{Resources}
   
   \begin{keysref}{Documentation}
-    Quantum \newline Development Kit & \url{https://docs.microsoft.com/quantum} \\
-    \qs~Language \newline Reference  & \url{https://docs.microsoft.com/quantum/language} \\
+    Quantum \newline Development Kit & \url{https://docs.microsoft.com/azure/quantum} \\
```

Comment:
Please don't modify this file or re-generate the quickref/qsharp-quick-reference.pdf that is generated from it. The setup for PDF generation is a bit tricky (I don't have it on my laptop), so I'm fine with not modifying it now - I noted this need and will get back to it when we're not in a rush.

Reply:
Understood, change reverted!